src/AspNet.Security.OpenId/OpenIdAuthenticationOptions.cs

```diff
@@ -12,13 +12,15 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.DataProtection;
 using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Authentication;
 using Microsoft.IdentityModel.Protocols;
 
 namespace AspNet.Security.OpenId
 {
     public class OpenIdAuthenticationOptions : RemoteAuthenticationOptions
     {
+		public string AuthenticationScheme { get; set; }
```

Comment:
Sure I mentioned that already, but those 2 properties need to disappear.

Reply:
there's no longer `AuthenticationScheme` and `DisplayName` in `RemoteAuthenticationOptions` nor `AuthenticationOptions `

Reply:
That's not how it works in 2.0. The scheme/display name is set by `builder.AddScheme()` and stored in `AuthenticationScheme`/`AuthenticationSchemeBuilder`.

Remove these 2 properties.

Reply:
@PinpointTownes Gotcha